blazor/diagram/connectors/segments/bezier/bezier.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: Bezier Segment in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about Bezier Segments in Syncfusion Blazor Diagram component and much more details.
+platform: Blazor
+control: Diagram Component
+documentation: ug
+---
+
+# Bezier segments in Blazor Diagram Component
+
+## How to create bezier segment
+
+Bezier segments are used to create curve segments and the curves are configurable either with the control points or with vectors. To create a bezier segment, the Type of the segment is set as [Bezier](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegmentType.html#Syncfusion_Blazor_Diagram_ConnectorSegmentType_Bezier) and need to specify type for the connector. The following code example illustrates how to create a default bezier segment.
```

Comment:
Bezier segments are used to create curve segments, and the curves are configurable either with the control points or vectors. To create a bezier segment, the Type of the segment is set as [Bezier](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegmentType.html#Syncfusion_Blazor_Diagram_ConnectorSegmentType_Bezier) and need to specify type for the connector. The following code example illustrates how to create a default bezier segment.